common/src/main/java/dev/schmarrn/lighty/core/LightyRenderer.java

```diff
@@ -112,49 +115,49 @@ private static int goThroughEachSection(Minecraft minecraft, Camera camera, Vec3
 
                 for (int ii = 0; ii < minecraft.level.getSectionsCount(); ++ii) {
                     SectionPos chunkSection = SectionPos.of(chunkPos, ii + minecraft.level.getMinSectionY());
-                    if (!minecraft.levelRenderer.isSectionCompiled(chunkSection.origin())) {
-                        // Don't bother doing anything if the chunk isn't rendered yet
+
+                    BufferHolder cachedBuffer = Compute.cachedBuffers.get(chunkSection);
+                    if (cachedBuffer == null) {
                         continue;
                     }
-
-                    if (Compute.cachedBuffers.containsKey(chunkSection)) {
-                        BufferHolder cachedBuffer = Compute.cachedBuffers.get(chunkSection);
-                        for (var entry : cachedBuffer.getGpuBuffers().entrySet()) {
-                            String key = entry.getKey();
-                            if (!cachedBuffer.isValid(key)) {
-                                continue;
-                            }
-                            if (!frustum.isVisible(
-                                    AABB.encapsulatingFullBlocks(chunkSection.origin().offset(-1, -1, -1), chunkSection.origin().offset(16, 16, 16))
-                            )) {
-                                continue;
-                            }
-                            // Only continue if the buffer is valid
-                            biggestBufferSize = addData(chunkSection, entry.getValue(), camPos, biggestBufferSize, drawList, transforms);
+                    for (var entry : cachedBuffer.getGpuBuffers().entrySet()) {
+                        ResourceLocation key = entry.getKey();
+                        if (!cachedBuffer.isValid(key)) {
+                            continue;
                         }
+                        if (!frustum.isVisible(
+                                AABB.encapsulatingFullBlocks(chunkSection.origin().offset(-1, -1, -1), chunkSection.origin().offset(16, 16, 16))
```

Comment:
Similar to previous question: If there is a benefit, it would also be good to change it here as well.

As long as the code stays consistent, either version is fine for me